core/src/font.rs

```diff
@@ -613,6 +619,9 @@ impl<'gc> Font<'gc> {
                 // Step horizontally.
                 transform.matrix.tx += twips_advance;
                 x += twips_advance;
+            } else {
+                // No glyph, zero advance
+                glyph_func(pos, &transform, &Glyph::empty(), Twips::ZERO, x);
```

Comment:
What's the difference between processing a "noop" glyph and skipping it entirely?

Reply:
I've described it in my commit message:

```
This is a natural assumption of Font::evaluate and makes writing code easier.
Code that renders glyphs will work the same way as non-renderable characters
will produce empty glyphs, but code that evaluates the font for measurement
purposes will use this property.
```

I'll add a short comment about it here too.

Reply:
Done